Number_types/include/CGAL/Gmpzf.h

```diff
@@ -165,10 +166,10 @@ INTERN_QUOTIENT::Real_embeddable_traits_quotient_base< Quotient<Gmpzf> >
 	  std::pair<std::pair<double, double>, long> d =
 	    q.denominator().to_interval_exp();
 
-	  CGAL_assertion_msg(CGAL::abs(1.0*n.second - d.second) < (1<<30)*2.0,
+	  CGAL_assertion_msg(CGAL::abs((double)(n.second - d.second)) < (1<<30)*2.0,
```

Comment:
See the discussion https://github.com/CGAL/cgal/pull/1035#issuecomment-213482117.